packages/custom-functions-metadata-plugin/package-lock.json

```diff
@@ -1,6 +1,10 @@
 {
   "name": "custom-functions-metadata-plugin",
+<<<<<<< HEAD
```

Comment:
You have unresolved merge conflicts in the package-lock files.

Reply:
Merge conflicts have been addressed